release/types.htaccess

```diff
@@ -8,18 +8,6 @@ Options +FollowSymlinks
 
 # Redirect /specification/java-annotation-design.html /jsr308/specification/java-annotation-design.html
 # Redirect /dev/specification/java-annotation-design.html /dev/jsr308/specification/java-annotation-design.html
-# Redirect /checker-framework/current/checkers-manual.html /checker-framework/current/checker-framework-manual.html
```

Comment:
I don't think you should remove this redirect.  Links on the Web may point to it.